src/client/components/additionalConfigs/modesThemeSwitcher/modesThemeSwitcher.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './modesThemeSwitcher.css';
+
+const ModesThemeSwitcher = () => {
+  return (
```

Comment:
You can avoid return statement in all components where you have nothing before this statement